test/apparmor.bats

```diff
@@ -129,7 +129,7 @@ run_a_container_after_unloading_default_apparmor_profile() {
 
 	pod_id=$(crictl runp "$TESTDIR"/apparmor5.json)
 
-	crictl create "$pod_id" "$TESTDIR"/apparmor_container5.json "$TESTDIR"/apparmor5.json && fail
+	run ! crictl create "$pod_id" "$TESTDIR"/apparmor_container5.json "$TESTDIR"/apparmor5.json
```

Comment:
it doesn't seem like this will ever fail. we need to check the status I think https://bats-core.readthedocs.io/en/stable/writing-tests.html#run-test-other-commands

Reply:
I should have kept reading:
```
    ! expect nonzero exit status (1-255), fail if command succeeds
```

:upside_down_face: